apis/chef/src/main/java/org/jclouds/chef/handlers/ChefErrorHandler.java

```diff
@@ -57,9 +57,7 @@ public void handleError(HttpCommand command, HttpResponse response) {
                exception = new AuthorizationException(message, exception);
                break;
             case 404:
-               if (!command.getCurrentRequest().getMethod().equals("DELETE")) {
-                  exception = new ResourceNotFoundException(message, exception);
-               }
+               exception = new ResourceNotFoundException(message, exception);
```

Comment:
@nacx Any comment on this change?

Reply:
Oh, I see that it is because of a test. I'd say the test is wrong, not this, so I'd just change the test and keep the current behavior.

Reply:
Done.